WooCommerce/Classes/ViewRelated/Blaze/ConfirmPayment/BlazeCampaignCreationErrorView.swift

```diff
@@ -0,0 +1,143 @@
+import SwiftUI
+
+/// View for the error state of a Blaze campaign creation request.
+struct BlazeCampaignCreationErrorView: View {
+    /// Scale of the view based on accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    @State private var isShowingSupport = false
+
+    private let onTryAgain: () -> Void
+    private let onCancel: () -> Void
+
+    init(onTryAgain: @escaping () -> Void,
+         onCancel: @escaping () -> Void) {
+        self.onTryAgain = onTryAgain
+        self.onCancel = onCancel
+    }
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: Layout.titlePadding) {
+                Image(uiImage: .bigErrorIcon)
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(width: Layout.errorIconSize * scale)
+                    .padding(.top, Layout.iconTopPadding)
+
+                Text(Localization.title)
+                    .bold()
+                    .largeTitleStyle()
+
+                VStack(alignment: .leading, spacing: Layout.contentPadding) {
```

Comment:
nit: The spacing for this VStack should be 12.

Reply:
Updated in 1f6f9f1de971737ce7514ee9e3c2d37f6296ba29.